src/pass.h

```diff
@@ -67,6 +67,25 @@ struct PassOptions {
   int optimizeLevel = 0;
   // 0, 1, 2 correspond to -O0, -Os, -Oz
   int shrinkLevel = 0;
+  // Function size at which we allways inline.
```

Comment:
I've been hoping to add as few as possible things to PassOptions, because otherwise it can grow without limit as more passes are added. However, a case can be made that inlining is so crucial that it does deserve to be there.

I'm unsure myself, curious to hear what other people think here.

Reply:
Ability of tuning inline's threshold limit and other params are great in my opinion. As I understand current default mechanics inlining only if size not grow significant after that but for `-O3`/`-O4` this not very important and much better if code will be faster, so in AssemblyScript for example we often apply manual inlining via decorators.

Reply:
Looking at the code it seemed the most logical place to put it, but I'd be happy to move it somewhere else.

If the concern is to avoid a large list of options where its not clear to which pass it belongs, maybe wrap it in an `InlineOptions` struct for grouping?